tests/unit/CMakeLists.txt

```diff
@@ -18,6 +18,17 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
-if (BUILD_DISPATCHER)
+if(BUILD_DISPATCHER)
```

Comment:
Can you, please, describe in the commit message how this is supposed to work? Please, list cmake configuration command and make command(s) to run tests.